app/cp_nigeria/views.py

```diff
@@ -200,6 +200,10 @@ def cpn_scenario_create(request, proj_id=None, step_id=STEP_MAPPING["choose_loca
             form = ProjectForm(request.POST)
             economic_data = EconomicProjectForm(request.POST)
         if form.is_valid() and economic_data.is_valid():
+            # the results could change without re-running the simulation if the exhcnage rate is changed, so we delete the report items if this is the case
+            qs_report = ImplementationPlanContent.objects.filter(simulation=project.scenario.simulation)
+            if qs_report.exists() and economic_data.has_changed():
+                qs_report.get().delete()
```

Comment:
If I am not mistaken, you don't need the `.get()` this pulls the data from DB to backend only to tell the DB to delete them, if you run `.delete()` alone it does it directly to the DB form the queryset